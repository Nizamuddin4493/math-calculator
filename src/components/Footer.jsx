import React from 'react';

function Footer() {
  return (
    <>
      <footer className="footer">
        <h4 className="bars"> / / </h4>
      </footer>
    </>
  );
}

export default Footer;
