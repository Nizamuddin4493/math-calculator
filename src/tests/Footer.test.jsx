import { BrowserRouter as Router } from 'react-router-dom';
import TestRenderer from 'react-test-renderer';
import Footer from '../components/Footer';

it('renders correctly', () => {
  const tree = TestRenderer.create(<Router><Footer /></Router>).toJSON();
  expect(tree).toMatchSnapshot();
});
