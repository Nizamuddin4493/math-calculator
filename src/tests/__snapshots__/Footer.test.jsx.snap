// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<footer
  className="footer"
>
  <h4
    className="bars"
  >
     / / 
  </h4>
</footer>
`;
